### PR TITLE
Modify IPU JAX notebooks to be compatible with Paperspace Gradient.

### PR DESCRIPTION
**Pull request by balancap**
Small modifications to the repository and notebook to add Paperspace compatibility:
  * `setup.sh` script to setup Paperspace IPU instance;
  * `pip install` commands to install JAX for IPUs;
  * README link to Paperspace;

### CODE REVIEW

File: ipu/examples/jax_ipu_quickstart.ipynb [hunk 1]
```diff
@@ -1,58 +1,56 @@
...
-   "execution_count": 1,
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install experimental JAX for IPUs\n",
+    "import sys\n",
+    "!{sys.executable} -m pip uninstall -y jax jaxlib\n",
+    "!{sys.executable} -m pip install https://github.com/graphcore-research/jax-mk2-experimental/releases/latest/download/jaxlib-0.3.15-cp38-none-manylinux2014_x86_64.whl\n",
+    "!{sys.executable} -m pip install https://github.com/graphcore-research/jax-mk2-experimental/releases/latest/download/jax-0.3.16-py3-none-any.whl"
```
Comment (awf) [inline ipu/examples/jax_ipu_quickstart.ipynb:21]: Great!

File: ipu/examples/jax_ipu_quickstart.ipynb [hunk 2]
```diff
@@ -1,58 +1,56 @@
...
    "metadata": {},
    "outputs": [],
    "source": [
     "from jax.config import config\n",
-    "# Using IPU model emulator. Comment to use IPU hardware.\n",
-    "config.FLAGS.jax_ipu_use_model = True\n",
-    "config.FLAGS.jax_ipu_model_num_tiles = 8"
+    "# Uncomment to use IPU model emulator.\n",
+    "# config.FLAGS.jax_ipu_use_model = True\n",
+    "# config.FLAGS.jax_ipu_model_num_tiles = 8"
```
Comment (awf) [inline ipu/examples/jax_ipu_quickstart.ipynb:33]: Yep!

File: ipu/examples/jax_ipu_quickstart.ipynb [hunk 3]
```diff
@@ -173,6 +136,34 @@
...
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### JAX Pseudo Random Numbers generation\n",
+    "\n",
+    "Reproducible random numbers across platforms using JAX ThreeFry PRNG."
```
Comment (awf) [inline ipu/examples/jax_ipu_quickstart.ipynb:146]: Is this already in?

**Review by awf — APPROVED** [review]

File: ipu/examples/jax_ipu_quickstart.ipynb (hunk 3 above)
Comment (balancap) [reply ipu/examples/jax_ipu_quickstart.ipynb:146]: The default XLA implementation of the ThreeFry algorithm is already working well on the IPU (following the fix to bitwise shift operators). Having a custom ThreeFry vertex will give us ~10x performance improvement (and faster compilation).

**Comment by balancap** [comment]
Thanks @awf !
